addons/account/tests/test_invoice_tax_totals.py

```diff
@@ -10,6 +10,7 @@ class TestTaxTotals(AccountTestInvoicingCommon):
     @classmethod
     def setUpClass(cls, chart_template_ref=None):
         super().setUpClass(chart_template_ref=chart_template_ref)
+        cls.maxDiff = None
```

Comment:
Please remove this :)

Reply:
I was just testing your review skills :roll_eyes:

Reply:
I'll do it as soon as the runbot is done :running_man:

Reply:
Done @casp-odoo